### PR TITLE
Convenience fixes

### PR DESCRIPTION
**Pull request by v1kko**

### CODE REVIEW

**Comment by v1kko** [comment]
Hi @stefsmeets , A lot of small changes/fixes in this one, but nothing major. If you still see something let me know!

File: src/duqtools/ids/_copy.py [hunk 1]
```diff
@@ -51,6 +52,45 @@ def get_imas_ual_version():
     return imas_version, ual_version
 
 
+def add_provenance_info(ids: ImasHandle):
+    """Add provenance information to ids.
+
+    Parameters
+    ----------
+    ids:
+        handle to add provenance information to
```
Comment (stefsmeets) [inline src/duqtools/ids/_copy.py:61]:
```suggestion
    ids: ImasHandle
        Handle to add provenance information to.
```

**Review by stefsmeets — APPROVED** [review]
Cool, nice work! 

Before your merge this, can you also call `add_provenance_info` at the end of `duqtools/merge.py`?

Actually, maybe we should call it here: https://github.com/CarbonCollective/fusion-dUQtools/blob/8c1b95c62bfddc5fb4bc2e4a997f395c153a30d7/src/duqtools/ids/_mapping.py#L84

File: src/duqtools/ids/_copy.py (hunk 1 above)
Comment (v1kko) [reply src/duqtools/ids/_copy.py]: Ah, yes I didn't have the merge.py file in this branch yet, and didn't see sync being used anywhere, but it definitely makes sense to add it there

**Pull request merged**